styles/atom-solidity.less

```diff
@@ -392,3 +392,13 @@
 .test-header {
     text-align: center;
 }
+
+button.btn.icon.icon-desktop-download.inline-block-tight.icon-button {
```

Comment:
we don't need this whole thing to style the button. I guess something like `.icon .icon-desktop-download` should be good enough.